operator/v1/0000_70_console-operator.crd.yaml

```
@@ -122,6 +122,11 @@ spec:
                                 type: array
                                 items:
                                   type: string
+                        disableTypes:
+                          description: disable types which are shown in the developer catalog.
+                          type: array
+                          items:
+                            type: string  
```

There are two spaces behind the type: string, so that the verify step doesn't pass:

```suggestion
                            type: string
```